app/src/main/res/values/colors.xml

```diff
@@ -84,6 +84,7 @@
     <color name="colorPrimaryOrange">#CE8500</color>
     <color name="colorPrimaryDandelionYellow">#F5BB00</color>
     <color name="colorPrimaryCoolBlue">#408cac</color>
+    <color name="colorPrimaryLavender">#9966CC</color>
```

Comment:
you need to add a lot more than just a color resource for this to work

Reply:
1. The style https://github.com/recloudstream/cloudstream/blob/master/app/src/main/res/values/styles.xml#L277
3. The code https://github.com/recloudstream/cloudstream/blob/93d81ea038ca3decc15b97d582c09eb8e4303c31/app/src/main/java/com/lagradost/cloudstream3/CommonActivity.kt#L321
4. The settings https://github.com/recloudstream/cloudstream/blob/93d81ea038ca3decc15b97d582c09eb8e4303c31/app/src/main/res/values/array.xml#L285